lib/transpose_cases.rb

```diff
@@ -0,0 +1,45 @@
+class TransposeCase < OpenStruct
+  def test_name
+    "test_#{description.tr(' ', '_')}"
+  end
+
+  def skipped
+    index.zero? ? '# skip' : 'skip'
+  end
+
+  def workload
+    'Transpose.transpose(input)'
+  end
+
+  def expect
+    [
+      "<<-EXPECTED.gsub(/^ {6}/, '')",
+      indent_lines(expected),
+      indent_line('EXPECTED')
```

Comment:
Consider that this is not critical.  So this should be something to think about, and play with.  The duplication here should not stop this PR.